core/src/main/scripts/importer/validateData.py

```diff
@@ -1250,15 +1250,7 @@ def printDataInvalidStatement(self, value, col_index):
 
 
     def checkNCBIbuild(self, value):
-        if value != '':
-            # based on MutationDataUtils.getNcbiBuild
-            if self.portal.species == "human":
-                if value not in [str(self.portal.ncbi_build), self.portal.genome_build, 'GRCh'+str(self.portal.ncbi_build)]:
-                    return False
-            elif self.portal.species == "mouse":
-                if value not in [str(self.portal.ncbi_build), self.portal.genome_build, 'GRCm'+str(self.portal.ncbi_build)]:
-                    return False
-        return True
+        return value in (self.portal.ncbi_build)
```

Comment:
I think I don't agree with this simplification of the code. Previous validation did check if the build was according to species expected. I don't think we support  multiple species yet in a single portal instance, so this check is still relevant. For mouse you need another set of entrez ids and another set of (possibly conflicting) hugo symbols. I think more work will be needed to support multiple same hugo symbols pointing to different entrez ids...I think some parts of the code try to come up with a single entrez for a given hugo...I think multi species might break this assumption.

Reply:
@pieterlukasse, no need, since 
if human, then we have
```
species=human
genome.build=37,hg19,GRCh37,GRCh38
genome.name=hg
```
if mouse, then we have
```
species=mouse
genome.build=GRCm37,GRCm38
genome.name=mm
```
do we have any use cases to support more than one species in single instance? if that is the case then we need something like below in the portal.properties file?
```
species=human,mouse
genome.build=37,hg19,GRCh37,GRCh38,GRCm37,GRCm38
genome.name=mam
```